app/src/main/java/com/teamfillin/fillin/NewPhotosAdapter.kt

```diff
@@ -0,0 +1,37 @@
+package com.teamfillin.fillin
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.teamfillin.fillin.databinding.ItemNewPhotosListBinding
+
+class NewPhotosAdapter : RecyclerView.Adapter<NewPhotosAdapter.NewPhotosViewHolder>() {
+
+    val photolist = mutableListOf<NewPhotosData>()
+
+    override fun onCreateViewHolder(
+        parent: ViewGroup,
+        viewType: Int
+    ): NewPhotosViewHolder {
+        val binding=ItemNewPhotosListBinding.inflate(
+            LayoutInflater.from(parent.context),
+            parent, false
+        )
+        return NewPhotosViewHolder(binding)
+    }
+
+
+    override fun onBindViewHolder(holder: NewPhotosViewHolder, position: Int) {
+        holder.onBind(photolist[position])
+    }
+    override fun getItemCount(): Int = photolist.size
+    class NewPhotosViewHolder(private val binding: ItemNewPhotosListBinding)
+        :RecyclerView.ViewHolder(binding.root){
+            fun onBind(data : NewPhotosData){
+                Glide.with(binding.root)
+                    .load(data.image)
+                    .into(binding.ivNewPhoto)
+            }
+        }
```

Comment:
```suggestion
    class NewPhotosViewHolder(
        private val binding: ItemNewPhotosListBinding
    ) : RecyclerView.ViewHolder(binding.root){
            fun onBind(data : NewPhotosData){
                Glide.with(binding.root)
                    .load(data.image)
                    .into(binding.ivNewPhoto)
            }
        }
```

일단 이런식으로 오른쪽으로 길어지는 코드는 안 만들 수 있음

또한 ImageView 같은 경우 ``binding.ivNewPhoto.load(data.image)``로도 작성할 수 있음(core 모듈 내에 ImageView 확장함수)

Reply:
좋네용